engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -4339,13 +4343,13 @@ private void orchestrateMigrateForScale(final String vmUuid, final long srcHostI
                     try {
                         _agentMgr.send(srcHostId, new Commands(cleanup(vm.getInstanceName())), null);
                     } catch (final AgentUnavailableException e) {
-                        s_logger.error("AgentUnavailableException while cleanup on source host: " + srcHostId);
+                        s_logger.error(String.format("AgentUnavailableException while cleanup on source host: %s", srcHost));
```

Comment:
@GabrielBrascher We can pass the exception as param to `s_logger.error` here.

Reply:
@GutoVeronezi that's a good option indeed, thanks for the attention. I've just updated it.